app.py

```
@@ -1072,6 +1115,63 @@ def track():
+@app.route('/api/payments/razorpay/order', methods=['POST'])
+def create_payment_order():
+    data = request.get_json(silent=True) or {}
+    try:
+        amount_rupees = float(data.get('amount', 0))
+    except (TypeError, ValueError):
+        return jsonify({'error': 'Invalid amount'}), 400
+
+    if amount_rupees < 10:
+        return jsonify({'error': 'Amount must be at least ₹10'}), 400
+
+    amount_paise = int(round(amount_rupees * 100))
```

Ruff (0.15.5)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)